messenger.rst

```diff
@@ -947,13 +1035,47 @@ holds them in memory during the request, which can be useful for testing.
 For example, if you have an ``async_priority_normal`` transport, you could
 override it in the ``test`` environment to use this transport:
 
-.. code-block:: yaml
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/test/messenger.yaml
+        framework:
+            messenger:
+                transports:
+                    async_priority_normal: 'in-memory:///'
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
```

Comment:
Should this be `config/packages/test/messenger.xml`?

Reply:
absolutely, thanks for the heads up 👍